vispy/ipython/widget.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2014, 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+from IPython.html.widgets import DOMWidget
+from IPython.utils.traitlets import Unicode, Int, Bool
+from vispy.app.backends._ipynb_util import create_glir_message
```

Comment:
> from vispy.ipython import VispyWidget

hmm I'd prefer `from vispy.app.backends.ipython import VispyWidget`

> maybe we can create the canvas directly from the widget or even make the
> widget derive from the widget ideas?

I don't understand can you explain?